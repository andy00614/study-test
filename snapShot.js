export const generateConfig = () => {
  return {
    server: 'http://localhost',
    port: 8088,
    time: new Date()
  }
}


export const generateConfig2 = () => {
  return {
    server: 'http://localhost',
    port: 8088,
    time: '20191'
  }
}