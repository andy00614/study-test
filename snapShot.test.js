import { generateConfig, generateConfig2 } from './snapShot'

test('test generateConfig',() => {
  expect(generateConfig()).toEqual({
    server: 'http://localhost',
    port: 8088
  })
})

// 用snapshot模式

test('test generateConfig snapShot',() => {
  expect(generateConfig()).toMatchSnapshot({
    time: expect.any(Date)
  });
})

test('test generateConfig snapShot2',() => {
  expect(generateConfig2()).toMatchSnapshot();
})