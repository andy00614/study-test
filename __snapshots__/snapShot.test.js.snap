// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test generateConfig snapShot 1`] = `
Object {
  "port": 8088,
  "server": "http://localhost",
  "time": Any<Date>,
}
`;

exports[`test generateConfig snapShot2 1`] = `
Object {
  "port": 8088,
  "server": "http://localhost",
  "time": "20191",
}
`;
